src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,44 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MAX_AGE = 120;
+    private static final int MIN_PASSWORD_LENGTH = 6;
 
     @Override
     public User register(User user) {
+        if (user == null) {
+            throw new RuntimeException("User must not be equal null");
+        }
+        if (user.getLogin() == null
+                || user.getAge() <= 0
+                || user.getAge() == null
+                || user.getPassword() == null
+                || user.getAge() >= MAX_AGE) {
+            throw new RuntimeException("Incorrect data");
+        }
+
+        StorageDao storageDao = new StorageDaoImpl();
```

Comment:
I think it is better to move storageDao to class fields. 
Firstly because you use it twice, and in my opinion, it is more like class quality, RegistrationService anyway means working with a db